src/track.c

```diff
@@ -395,8 +462,12 @@ void tracking_channel_update(u8 channel)
       break;
     }
     case TRACKING_DISABLED:
+      /* TODO: Why do we sometimes have to disable the channel in the
+               NAP repeatedly? */
```

Comment:
Yeah I tried that but it was spamming every time a tracking channel got disabled.  I think it's because the FPGA may still do one more update.